quic/api/Observer.h

```diff
@@ -113,6 +129,16 @@ class InstrumentationObserver {
   virtual void rttSampleGenerated(
       QuicSocket*, /* socket */
       const PacketRTT& /* RTT sample */) {}
+
+  /**
+   * rttSampleGenerated() is invoked when a RTT sample is made.
+   *
+   * @param socket   Socket when the callback is processed.
+   * @param packet   const reference to the packet spuriously retransmitted.
```

Comment:
There's no guarantee it was retransmitted -- just lost. I think there may be something in the metadata that would hint if it was retransmitted.

Reply:
Also packets in QUIC aren't retransmitted ;). Data can be retransmitted.